modules/osdk-cli-reference-generate-packagemanifests.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-packagemanifests_{context}"]
+= packagemanifests
+
+Running `generate packagemanifests` subcommand is the first step to publishing your operator to a catalog and/or deploying
+it with OLM. This command generates a set of manifests in a versioned directory and a package manifest file for
+your operator. You must run `generate kustomize manifests` first to regenerate kustomize bases
+consumed by this command.
+
+
+.`generate packagemanifests` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`--channel` (string)
+|The channel name for the generated package.
+
+|`--crds-dir` (string)
+|The root directory for Custom Resource Definition (CRD) manifests.
```

Comment:
I'm sure that the docs have the C, R, and D upcased endlessly, but they should be lowercase, as described in the glossary entry for [custom resource definition](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#custom-resource-definition-crd).  Please lowercase.

Reply:
I, too, wasn't sure about this. Thanks for clarifying.